### ISSUE TITLE
Write standalone uart backend application for oRSC

### ISSUE THREAD

**Issue by nwoods** [uwcms/cms-calo-layer1#43]
Without Linux on oRSC-BE, we need standalone code to receive uart signals from FE. First, we are trying to replicate orsc_fe_spi_echo_test but with uart.

**Comment by nwoods** [comment]
People who should be on this thread: @tsarangi, @dabelknap, @mathiasblake, @ekfriis. Did I forget anyone?

**Comment by mathiasblake** [comment]
Nate,

This is incorrect.

-Mathias

**Comment by nwoods** [comment]
Edited. Is it good now?

**Comment by mathiasblake** [comment]
yes.  Just need to remove the part about sending stuff out over links.
